src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import './utils/_variables.scss';
+@import './utils/_functions.scss';
+@import './utils/_mixins.scss';
+
+@import './blocks/page.scss';
+@import './blocks/calendar.scss';
```

Comment:
You can change single quotation marks ' ' with doubles quotation marks " " to pass the npm tests.